man/systemd.network.xml

```diff
@@ -1903,6 +1903,16 @@
         <varname>DNS=</varname>.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>LogServer=</varname></term>
+
+        <listitem><para>Similar to the <varname>DNS=</varname> settings
+        described above, these settings configure whether and what log server
```

Comment:
Hmm, maybe add a short explanation what "log server" is? While "DNS" and "POP3" and "SMTP" are all well known terms, "log" is rather obscure.